.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Description: 
+Summarize the changes you're submitting in a few sentences, including Jira ticket ATL-xxxx if applicable.
+Link to any discussion, related issues and bug reports to give the context to help the reviewer understand the PR.
```

Comment:
Link to any discussion, related issues, and bug reports to give the context and help the reviewer understand the PR.

Reply:
@petevielhaber , I have added your comment to take into account in the issue #254. If ok, please resolve the conversation.